python/IECoreDelightTest/ShaderNetworkAlgoTest.py

```diff
@@ -517,5 +517,44 @@ def testConvertUSDPrimvarReader( self ) :
 			self.assertEqual( reader.parameters["attribute_name" if convertedShaderType == "dlPrimitiveAttribute" else "name"].value, "test" )
 			self.assertEqual( reader.parameters["fallback_value" if convertedShaderType == "dlPrimitiveAttribute" else "defaultValue"].value, convertedDefault )
 
+	def testConvertUSDUVTextureUDIM( self ) :
+
+		for uvPrimvar in ( "st", "customUV" ) :
```

Comment:
I can't see the different values of `uvPrimvar` having any real influence on the test, or being likely to trip up in `convertUSDShaders` in a way that would show up. Am I missing something?

Reply:
Nope, this is totally unnecessary. Simplified in aa17cc7.